docs/source/app-dev/bindings-scala/index.rst

```diff
@@ -0,0 +1,144 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Scala bindings
+##############
+
+This tutorial provides a basic Scala programmer's introduction to working with Digital Asset distributed ledger,
+using the Scala programming language and the *Ledger API*.
+
+Introduction
+============
+
+The Scala bindings is a client implementation of the *Ledger API*. The Scala bindings library lets you write applications that connect to the Digital Asset distributed ledger using the Scala programming language.
+
+There are two main components:
+
+- Scala codegen
+    DAML to Scala code generator. Use this to generate Scala classes from DAML models. The generated Scala code provides a type safe way of creating contracts (:ref:`com.digitalasset.ledger.api.v1.createcommand`) and exercising contract choices (:ref:`com.digitalasset.ledger.api.v1.exercisecommand`).
+
+- Akka Streams-based API
+    The API that you use to send commands to the ledger and receive transactions back.
+
+In order to use the Scala bindings, you should be familiar with:
+
+- :doc:`DAML language </daml/reference/index>`
+- :doc:`Ledger API </app-dev/grpc/index>`
+- `Akka Streams API <https://doc.akka.io/docs/akka/current/stream/index.html>`_
+- `Scala programming language <https://www.scala-lang.org>`_
+- :ref:`assistant-manual-building-dars`
+
+Getting started with sbt
```

Comment:
I'm a bit confused as to what you're doing here. Are you telling people how to do something, or are you showing them what an example build.sbt file looks like? You need to explicitly tell the reader what the purpose of the information you're giving them is.